sycl/doc/extensions/proposed/sycl_ext_codeplay_cuda_cluster_group.asciidoc

```diff
@@ -0,0 +1,420 @@
+= sycl_ext_codeplay_cuda_cluster_group
+
+:source-highlighter: coderay 
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+:endnote: &#8212;{nbsp}end{nbsp}note
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+
+== Notice
+
+[%hardbreaks] 
+
+Copyright (C) 2024-2024 Intel Corporation.  All rights reserved.
```

Comment:
```suggestion
Copyright (C) 2024-2024 Codeplay Corporation.  All rights reserved.
```
If this is a Codeplay extension then should be Codeplay (C)

Reply:
Thanks for catching this.
Committing this verbatim